### DISCUSSION TITLE
Automatically add date and time

### DISCUSSION THREAD

**Discussion by pgyogesh** [mgmeyers/obsidian-kanban#601] [Ideas & Feature Requests]
I would like to have date and time automatically inserted when I create a card. May be through setting if not by default/

**Comment by GBergatto** [comment]
I'm interested in this feature, too.
I'd like the plug-in to automatically add a due date anytime I add a card to a column or move a card from one column to another. Ideally, there would be a section in the board settings where I can set the amount of time before the due date specific for each column.

Here's a screenshot of this feature inside Trello
![image](https://user-images.githubusercontent.com/69045457/193422437-8eacee3c-5132-4db7-95e6-dc4c9e3badba.png)

**Comment by magpie514** [comment]
I'm interested in this too. The adding a date/time on card creation, I mean.

Is there any other way to achieve this in the meantime?
